pkg/utils/image/infos.go

```diff
@@ -39,7 +45,7 @@ func (i *ImageInfo) String() string {
 	}
 }
 
-func (i *ImageInfo) ReferenceWithTag() string {
+func (i *ImageInfo) GetReferenceWithTag() string {
```

Comment:
Why is this public function needed at all now? It can just be moved to the function below IMO.